src/flows/pipes/RawFluxEditor/index.ts

```diff
@@ -33,11 +32,9 @@ export default register => {
         const ast = parse(
           data.queries[data.activeQuery].text.replace(PREVIOUS_REGEXP, query)
         )
-        find(ast, node => !!Object.keys(node.comments || {}).length).forEach(
-          node => {
-            delete node.comments
-          }
-        )
+        if (!ast.body.length) {
+          return ''
+        }
```

Comment:
a change must have been made in the wasm because now `parse` handles comments differently. `parse` removes them automatically and the property `.comments` no longer exists.

Reply:
I don't think the distinction matters for this pr, but in case we have to do something with comments or query comparison later, are you saying that a valid query, with a comment and some flux, when passed through parse, then through format, will have the comments removed?

Reply:
after more testing, it looks like I was wrong. We still do need to parse comments. Reverted this change